java-kotlin-aot/README.md

```diff
@@ -12,44 +12,27 @@ This repository contains the code for a demo application for [GraalVM](graalvm.o
 This is a simple Java/Kotlin application, where a Java method accesses a String from Kotlin and calls a Kotlin function, which later accesses a String from a Java class.
 This example demonstrates how easy it is to interop between Java and Kotlin.
 
```

Comment:
In prerequisites, correct the link to NI documentation:

https://www.graalvm.org/docs/reference-manual/native-image/
to
https://www.graalvm.org/latest/reference-manual/native-image/